c4c-internal-backend-master/api/target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst

```diff
@@ -0,0 +1,5 @@
+C:\Users\Andrew Lucia\Documents\C4C\c4c-internal-backend-master\api\src\main\java\com\codeforcommunity\dto\MemberReturn.java
```

Comment:
please remove this file and as well as extraneous binaries and jars. Artifacts and as well as machine specific files should not be shared on git with src code.